app/backend/models/trading-eq-follow.js

```diff
@@ -0,0 +1,17 @@
+const {mongoose} =  require('../db');  // The mongodb connector library
+
+let TradingEquipmentFollow = mongoose.model('TradingEquipmentFollow', {
+  UserId: {
+    type: String,
+    require: true
+  },
+
+  TradingEq: {
+    type: String,
```

Comment:
We might use the type ObjectId here as well.

Reply:
no need for ObjectId